src/app/access-requests/features/access-request-dialog/access-request-dialog.component.html

```diff
@@ -37,7 +37,7 @@ <h2 mat-dialog-title>
         [max]="maxUntilDate"
         [(ngModel)]="untilDate"
       />
-      <mat-hint>MM/DD/YYYY</mat-hint>
+      <mat-hint>DD/MM/YYYY</mat-hint>
```

Comment:
So the output would be ISO and the input European style? Shouldn't we also use ISO for input?

Reply:
I'm not sure about that. On a German keyboard, hyphen and dot are easier to type than slash. Also either DD.MM.YYYY or YYYY-MM-DD (ISO) is the preferred format in official EU documents.